### DISCUSSION TITLE
Support for textarea component

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#3246] [Ideas]
Currently under the form list, there is only input for type "text"
It would be good to have type "textarea" for multi-line data.

**Comment by mjashanks** [comment]
Totally agree. We have an open issue for this: https://github.com/Budibase/budibase/issues/2967

Currently, the "Rich Text" component displays a simple text area! But it's not very nice to use as it's not resizable. This is something we really need to improve.

**Comment by melohagan** [comment]
We now have the [Long form field](https://docs.budibase.com/docs/text-inputs#long-form-field) !
